fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/AbstractNonRdfSourcePersister.java

```diff
@@ -108,7 +108,8 @@ protected void persistNonRDFSource(final ResourceOperation operation,
         }
 
         // Write resource headers
-        final var headers = populateHeaders(objectSession, subpath, nonRdfSourceOperation, outcome);
+        final var headers = populateHeaders(objectSession, subpath, nonRdfSourceOperation, outcome,
+                fedoraSubpath == "");
```

Comment:
It would be good to have a test somewhere that verifies this new behavior, maybe in CreateNonRdfSourcePersisterTest. An integration test would be nice since it'd make sure that a blank subpath gets generated, but that might not be practical at the moment, not sure.

Reply:
I'll take a look.

Reply:
A unit test explicitly covers it here: https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-persistence-ocfl/src/test/java/org/fcrepo/persistence/ocfl/impl/OCFLPersistentStorageUtilsTest.java#L91.